test/test-os-version.js

```diff
@@ -25,6 +25,8 @@ if (common.isWindows()) {
   tap.match(version_str,
             new RegExp('OS version: ' + os_name + ' \\d+.' + os.release()),
             'Checking OS version');
+} else if (common.isSunOS()) {
```

Comment:
Cross referencing this with my earlier comment at https://github.com/nodejs/node-report/pull/62#issuecomment-281434043. I believe we should be able to _not_ skip this test.